src/content/configuration/module.mdx

```diff
@@ -49,6 +49,22 @@ module.exports = {
 };
 ```
 
+## `module.unsafeCache`
+
+`boolean: false` `function()`
```

Comment:
Formatting is a bit off, also did you mean `false` by default or only `false` is allowed when boolean given?
```suggestion
`boolean: false` `function ()`
```

Reply:
I meant false by default

Reply:
ok in that case it should be:
```suggestion
`boolean = false` `function ()`
```